30/feoh/pompom.py

```diff
@@ -0,0 +1,108 @@
+import json
+import argparse
+import time
+import sys
+import readchar
+from logbook import RotatingFileHandler, info, notice, debug, StreamHandler
```

Comment:
pep8: order of imports

Reply:
cool, have not used `logbook` yet

Reply:
It's really, *really* nice! I wish the documentation were better, but maybe I'll have the chance to contribute and improve that someday :)

For example, all you need is these two lines to set up an automatically rotated log file and simultaneously log to stdout:

```
    RotatingFileHandler("Pompom.log").push_application()
    StreamHandler(sys.stderr, level='INFO', bubble=True).push_application()
```

Pretty neat :)